### PR TITLE
Fix #10 - add options and catch the boolean error

### LINKED ISSUE #10: test
test

### PR DESCRIPTION
**Pull request by microdataxyz**
@michalvankodev I changed the followings:
* use the new numbers for the readme. It supposes to be v1.2.1. Tell me if I am wrong.
* add a new input to the action.yaml file.
* create a `getBooleanInput` function with the fallback value
* don't call the API if no labels found

I did try to update the version at the github workflow actions. It failed because the version isn't published

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -8,6 +8,8 @@ inputs:
     description: 'The GITHUB_TOKEN secret'
   custom-keywords:
     description: 'List of custom keywords which should be included in the parser'
+  from-title:
+    description: 'To parse the issue numbers from title, a boolean'
```
Comment (michalvankodev) [inline action.yml:12]:
```suggestion
  from-title:
    description: 'To parse the issue numbers from title, a boolean'
    required: false
```

This parameter should be optional

File: src/index.ts [hunk 2]
```diff
@@ -66,7 +74,7 @@ async function run() {
     }, [])
   )
 
-  await client.issues.addLabels({
+  labels.length > 0 && await client.issues.addLabels({
```
Comment (michalvankodev) [inline src/index.ts:77]: Thank you for this !

**Review by michalvankodev — COMMENTED** [review]
To be able to test this action it has to be "released". That's on me. First I'll need to add a git tag to the commit and then release it within the github platform.

File: action.yml (hunk 1 above)
Comment (microdataxyz) [reply action.yml:12]: Nice catch..

**Comment by microdataxyz** [comment]
@michalvankodev Please review the pr again.. Thanks...

**Review by michalvankodev — APPROVED** [review]

**Pull request merged**